src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -151,6 +151,33 @@ protected SemanticModel GetSemanticModel(SyntaxTree tree)
 
                 AddOverloadCountPart(symbols);
                 FixAllAnonymousTypes(symbols[0]);
+                AddExceptions(symbols[0]);
+            }
+
+            private void AddExceptions(ISymbol symbol)
+            {
+                // clean up the list of possible exceptions by de-duplicating and ordering them
+                var exceptions =
+                    symbol.GetDocumentationComment().ExceptionTypes
+                    .Distinct()
+                    .OrderBy(e => e)
```

Comment:
That's a good point. Distinct itself is going to do a ordinal filter - which is correct. 

Given this is for display purposes, you should pass in StringComparer.CurrentCultureIgnoreCase OrderBy to order them in the current culture's sort order.